README.md

```diff
@@ -1,6 +1,9 @@
 # Introduction
 This repo contains evaluation metric codes used in Microsoft Cognitive Services Computer Vision for tasks such as classification, object detection, and image caption.
 
+## Requirements ##
+- Java 1.8.0
```

Comment:
Not all evaluators requires Java 1.8.0. Could you make it more specific? Thanks

Reply:
fixed!